File: src/main/java/xyz/gnarbot/gnar/commands/executors/general/UptimeCommand.java
package xyz.gnarbot.gnar.commands.executors.general;

import xyz.gnarbot.gnar.Bot;
import xyz.gnarbot.gnar.commands.handlers.Command;
import xyz.gnarbot.gnar.commands.handlers.CommandExecutor;
import xyz.gnarbot.gnar.utils.Note;

@Command(aliases = "uptime", description = "Show the bot's uptime.")
public class UptimeCommand extends CommandExecutor
{
    @Override
    public void execute(Note note, String label, String[] args)
    {
        note.replyEmbedRaw("Bot Uptime", Bot.INSTANCE.getUptime());
    }
}

File: src/main/kotlin/xyz/gnarbot/gnar/commands/executors/general/UptimeCommand.kt
package xyz.gnarbot.gnar.commands.executors.general

import net.dv8tion.jda.core.entities.Message
import xyz.gnarbot.gnar.Constants
import xyz.gnarbot.gnar.commands.handlers.Command
import xyz.gnarbot.gnar.commands.handlers.CommandExecutor

@Command(aliases = arrayOf("uptime"), description = "Show the getBot's uptime.")
class UptimeCommand : CommandExecutor() {
    override fun execute(message: Message, args: Array<String>) {
        val s = bot.uptime / 1000
        val m = s / 60
        val h = m / 60
        val d = h / 24

        message.respond().embed("Bot Uptime") {
            color = Constants.COLOR
            description = "$d days, ${h % 24} hours, ${m % 60} minutes and ${s % 60} seconds"
        }.rest().queue()
    }
}
